lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -99,7 +107,12 @@ function _signup_resource_access() {
  * @param boolean $competition
  * @return array
  */
-function _signup_resource_index($user, $campaigns, $count, $page, $competition) {
+function _signup_resource_index($user, $users, $campaigns, $count, $page, $competition) {
+
+  if (isset($users)) {
+    $user = $users;
```

Comment:
Oh yeah, I got that, but eventually we should change those lines of code in other places so it doesn't lead to confusion.